generate_podspec.sh

```diff
@@ -13,11 +13,15 @@ do
 done
 
 echo $VERSION
-
 if [ -z $VERSION ]; then
     echo 'Version is required'
 fi
 
+echo $SHA1
+if [ -z $SHA1 ]; then
+    echo 'shasum is required'
+fi
```

Comment:
exit 1 추가해도 좋을 것 같아요!

```suggestion
if [ -z $VERSION ]; then
    echo 'Version is required'
    exit 1
fi

echo $SHA1
if [ -z $SHA1 ]; then
    echo 'shasum is required'
    exit 1
fi
```